src/routes/SelectRepos/index.js

```diff
@@ -82,6 +82,20 @@ export class SelectRepos extends React.Component {
           Can&#8217;t find one of your repos here? Make sure you are watching it
           on Github!
         </p>
+        {this.props.selectedRepos.length ? (
```

Comment:
a nicer pattern is to do
```
{ this.props.selectedRepos.length && < [... your JSX] /> }
```
I haven't been doing this in the project so far but want to shift to it, means you don't need to return null or undefined when condition not met.

Reply:
I wanted to do this but some linters don't like it. I can change it.